<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="kmaster" failures="0" tests="4" name="test.BasicValidation1" time="0.015" errors="1" timestamp="2025-08-02T17:12:04 GST" skipped="3">
  <testcase classname="test.BasicValidation1" name="welcomeMessageValidation" time="0.000">
    <skipped/>
  </testcase> <!-- welcomeMessageValidation -->
  <testcase classname="test.BasicValidation1" name="emailTextBoxPlaceholderValidation" time="0.000">
    <skipped/>
  </testcase> <!-- emailTextBoxPlaceholderValidation -->
  <testcase classname="test.BasicValidation1" name="setup" time="0.015">
    <error message="Cannot invoke &quot;test.BasicValidation1.setup()&quot; because the return value of &quot;io.github.bonigarcia.wdm.WebDriverManager.chromedriver()&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "test.BasicValidation1.setup()" because the return value of "io.github.bonigarcia.wdm.WebDriverManager.chromedriver()" is null
at test.BasicValidation1.setup(BasicValidation1.java:23)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:657)
at org.testng.TestRunner.beforeRun(TestRunner.java:646)
at org.testng.TestRunner.run(TestRunner.java:612)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- setup -->
  <testcase classname="test.BasicValidation1" name="passwordBoxPlaceholderValidation" time="0.000">
    <skipped/>
  </testcase> <!-- passwordBoxPlaceholderValidation -->
</testsuite> <!-- test.BasicValidation1 -->
